setup.cfg

```diff
@@ -19,23 +19,12 @@ data-files =
     share/ansible-runner/utils = utils/*
 
 [pep8]
-# E201 - Whitespace after '('
-# E203 - Whitespace before ":"
-# E221 - Multiple spaces after operator
-# E225 - Missing whitespace around operator
-# E231 - Missing whitespace after ','
-# E241 - Multiple spaces after ','
-# E251 - Unexpected spaces around keyword / parameter equals
-# E261 - At least two spaces before inline comment
-# E302 - Expected 2 blank lines found 0
-# E303 - Too many blank lines
-# W291 - Trailing whitespace
-# W391 - Blank line at end of file
-# W293 - Blank line contains whitespace
-ignore=E201,E203,E221,E225,E231,E241,E251,E261,E265,E303,W291,W391,W293
+# W503 - Line break occurred before a binary operator
+ignore=W503
 exclude=.tox,venv
 
 [flake8]
+# W503 - Line break occurred before a binary operator
+ignore=W503
```

Comment:
This shouldn't be necessary because this rule is already listed in the default `ignore` setting — https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-ignore:
```suggestion
```

Moreover, it's best to avoid using the `ignore` setting because it replaces what flake8 has set there by default (`E121,E123,E126,E226,E24,E704,W503,W504`).

Instead, if you want to add new ignore entries w/o replacing that underlying ignore list, use `extend-ignore` — https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-extend-ignore

Reply:
Personally, I do not like the default ignore list and would prefer we override `ignore` here, as this change is doing.

Reply:
I didn't realize that `ignore` overwrote the default list. I thought the behavior of `extend-ignore` was the behavior of `ignore`. That explains why I've been confused by these two rules in the past.

Reply:
After looking at the rules in the default ignore list, I'm inclined to enable them all and only ignore W503 as we have here. I accidentally did what I like. :)